app_dart/lib/src/request_handlers/get_release_branches.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:github/github.dart';
+
+import '../../cocoon_service.dart';
+
+/// Return a list of commit shas of the latest 5 branches for google3 roll, beta, and stable.
+///
+/// Branches are sorted based on the version number.
```

Comment:
I would avoid making guarantees like this in the docs if its not critical.

Reply:
Sure, removed.